addons/viewport/src/Tool.tsx

```diff
@@ -196,10 +196,10 @@ export const ViewportTool: FunctionComponent = memo(
             <Global
               styles={{
                 [`#${iframeId}`]: {
-                  margin: `auto`,
+                  margin: `${theme.layoutMargin}px auto`,
```

Comment:
I think this breaks being able to view the entire iframe when the iframe is bigger then then preview area:

<img width="1919" alt="Screenshot 2019-11-15 at 01 48 48" src="https://user-images.githubusercontent.com/3070389/68908006-19cc1180-074a-11ea-9af7-25b5082d65b8.png">

Reply:
I'll revert that change 👍